scripts/LocationList.js

```diff
@@ -27,7 +40,7 @@ function LocationList() {
     // Iterate through allLocations using a for..of loop
     for (const location of allLocations) {
         // Get services for the current park area
-        const servicesList = getServicesForParkArea(location.id).join(", ");
+        const servicesList = getLocationServiceMatches(location.id, allLocationServices, allServices).join(", ");
```

Comment:
When you call the function, you don't need .id after location, if you just put in location it should loop through them all and will get each id that way